test/spec/smartbanner_spec.js

```diff
@@ -224,6 +225,76 @@ describe('SmartBanner', function() {
 
     });
 
+    context('when enabledPlatform set to android, but is openned on a iPhone', function() {
+      const HTML_WITH_PLATFROM_OPTION_ANDROID = `<!doctype html>
+        <html style="margin-top:10px;">
+        <head>
+          <meta charset="utf-8">
+          <meta name="smartbanner:title" content="Smart Application">
+          <meta name="smartbanner:author" content="SmartBanner Contributors">
+          <meta name="smartbanner:price" content="FREE">
+          <meta name="smartbanner:price-suffix-apple" content=" - On the App Store">
+          <meta name="smartbanner:price-suffix-google" content=" - In Google Play">
+          <meta name="smartbanner:icon-apple" content="icon--apple.jpg">
+          <meta name="smartbanner:icon-google" content="icon--google.jpg">
+          <meta name="smartbanner:button" content="View">
+          <meta name="smartbanner:button-url-apple" content="https://itunes.apple.com/us/genre/ios/id36?mt=8">
+          <meta name="smartbanner:button-url-google" content="https://play.google.com/store">
+          <meta name="smartbanner:enabledPlatforms" content="android">
+        </head>
+        <body>
+        </body>
+      </html>`;
+
+      before(function() {
+        global.window = jsdom.jsdom(HTML_WITH_PLATFROM_OPTION_ANDROID, { userAgent: USER_AGENT_IPHONE }).defaultView;
+        global.document = window.document;
+        global.getComputedStyle = window.getComputedStyle;
+        smartbanner = new SmartBanner();
+        smartbanner.publish();
+      });
+
+      it('expected to not add anything to body', function() {
+        expect(document.querySelector('.js_smartbanner')).to.be.null;
+      });
+
+    });
+
+    context('when enabledPlatform set to ios, but is openned on a Android', function() {
```

Comment:
Let's improve here a bit, e.g. `when enabled-platform set to ios, but opened on Android`